packages/gestalt/src/Upsell.js

```diff
@@ -0,0 +1,164 @@
+// @flow strict
+import React, { type Node } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Box from './Box.js';
+import Heading from './Heading.js';
+import IconButton from './IconButton.js';
+import Button from './Button.js';
+import Text from './Text.js';
+import { type AbstractEventHandler } from './AbstractEventHandler.js';
+import styles from './Upsell.css';
+
+type LinkData = {|
+  accessibilityLabel?: string,
+  href: string,
+  label: string,
+  onClick?: AbstractEventHandler<
+    | SyntheticMouseEvent<HTMLButtonElement>
+    | SyntheticMouseEvent<HTMLAnchorElement>
+    | SyntheticKeyboardEvent<HTMLAnchorElement>
+    | SyntheticKeyboardEvent<HTMLButtonElement>
+  >,
+|};
+
+type Props = {|
+  dismissButton?: {|
+    accessibilityLabel: string,
+    onDismiss: () => void,
+  |},
+  message: string,
+  primaryLink?: LinkData,
+  secondaryLink?: LinkData,
+  title?: string,
+|};
+
+const UpsellLink = ({
+  data,
+  stacked,
+  type,
+}: {|
+  data: LinkData,
+  stacked?: boolean,
+  type: string,
+|}): Node => {
+  const color = type === 'primary' ? 'red' : 'gray';
+  const { accessibilityLabel, href, label, onClick } = data;
+
+  return (
+    <Box
+      display="flex"
+      alignItems="center"
+      justifyContent="center"
+      paddingX={1}
+      marginTop={type === 'primary' && stacked ? 2 : undefined}
+      mdMarginTop="auto"
+      mdMarginBottom="auto"
+      rounding={type === 'primary' ? 'pill' : undefined}
```

Comment:
we don't need this line. It's probably dead code from before having LinkButtons

Reply:
Removed! And also removed from `Callout.js`

Reply:
Thank you!